trading_calendars/common_holidays.py

```diff
@@ -0,0 +1,87 @@
+from pandas.tseries.holiday import Holiday, Easter
+from pandas.tseries.offsets import Day
+
+from .trading_calendar import MONDAY, TUESDAY
+
+
+def new_years_day(start_date=None,
```

Comment:
API-wise it seems a bit confusing to have a mix of factory functions and holiday objects in this file. What do you think about just having a consistent interface of:

```
def snake_case_holiday_name(start_date, end_date, offset, observance):
    return Holiday(...)
```
?

Reply:
forgot to push this friday, just pushed